src/Razor/src/Microsoft.AspNetCore.Razor.Language/DefaultRazorIntermediateNodeLoweringPhase.cs

```diff
@@ -777,7 +813,6 @@ public override void VisitMarkupTextLiteral(MarkupTextLiteralSyntax node)
                     if (existingHtmlContent.Source == null && source == null)
                     {
                         Combine(existingHtmlContent, node);
-                        base.VisitMarkupTextLiteral(node);
```

Comment:
Did all of these base.XYZ removals just noop before?

Reply:
Yup